docs/Advanced-Concepts/Architectural-Checkpoints.rst

```diff
@@ -29,10 +29,33 @@ Loading Checkpoints in RTL Simulation
 --------------------------------------
 
 Checkpoints can be loaded in RTL simulations with the ``LOADARCH`` flag.
-The target config **MUST** use dmi-based bringup (as opposed to the default TSI-based bringup), and support fast ``LOADMEM``.
-The target config should also match the architectural configuration of however spike was configured when generating the checkpoint.
+The target config needs the following properties:
+
+- **MUST** use DMI-based bringup (as opposed to the default TSI-based bringup)
+- **MUST** support fast ``LOADMEM``
+- Should match the architectural configuration of however Spike was configured when generating the checkpoint (i.e. same ISA, no PMPs, etc).
 
 .. code::
 
    cd sims/vcs
-   make CONFIG=dmiRocketConfig run-binary LOADARCH=../../hello.riscv.0x80000000.1000.loadarch
+   make CONFIG=dmiRocketConfig run-binary LOADARCH=../../hello.riscv.*.loadarch
+
+Checkpointing Linux Binaries
+----------------------------
+
+Checkpoints can be used to run Linux binaries with the following caveats:
+
+- The binary must only use the HTIF console and should be non-interactive (i.e no stdin available)
+- The target config must be built without a serial device (i.e. the Rocket Chip Blocks UART can't be used)
+- The binary must only use an initramfs (i.e. no block device)
+- The target config must be built without a block device (i.e. the IceBlk block device can't be used).
```

Comment:
RamFS + OpenSBI must be smaller than the size of target memory (so if ramfs in FM is set to 1G, you should set it to 1G + 1M (opensbi is under 1M))